README.md

```diff
@@ -25,6 +25,7 @@
 
 </div>
 
+- [**Documentation**](#documentation)
```

Comment:
i don't think it is so important to go first, let's mover it after feedback

Reply:
> i don't think it is so important to go first, let's mover it after feedback, the same about content

Sure, I moved it after `feedback` for both the outline and actual content.